### PR TITLE
Fixes for second chapter of docker guide

### PR DESCRIPTION
**Pull request by wronawiktor**

### CODE REVIEW

File: docs/containers/create-first-webapp.md [hunk 1]
```diff
@@ -42,11 +42,15 @@ Start a container with the `hello-world` website:
...
+The `hello-world` page is now available at [http://127.0.0.1:8081](http://127.0.0.1:8081)
 
 Test the `hello-world` website using the `curl` command:
 
 ```shell
 curl http://127.0.0.1:8081
 ```
+Aftrer this you can kill container using:
+```shell
+docker stop (<container_ID> or <container_name>)
```
Comment (mjura) [inline docs/containers/create-first-webapp.md:54]: Please add example how to get container ID using `docker ps` command $(docker ps -q -f name=<CONTAINER_NAME>)

Comment (mjura) [inline docs/containers/create-first-webapp.md]:
```suggestion
Stop container using name

docker stop webapp

or using container ID

docker stop $(docker ps -q -f name=webapp)
```

**Review by mjura — CHANGES_REQUESTED** [review]
Please add more hints howto stop container

**Comment by mjura** [comment]
please rebase

**Comment by wronawiktor** [comment]
test

**Comment by wronawiktor** [comment]
test2

**Review by mjura — APPROVED** [review]

**Pull request merged**